### ISSUE TITLE
Gangs & Turf wars

### ISSUE THREAD

**Issue by thecodeah** [opencnr/gamemode#14]

**Comment by ADRFranklin** [comment]
I'm not sure if this fits in with the CnR theme. Maybe this could be some kind of spin off event mode or something, that could be active once or twice a month,

**Comment by thecodeah** [comment]
Yeah, that's true. I'm just wondering what other game features could be added aside from just robbing and arresting.
